src/components/ui/datefield.tsx

```diff
@@ -0,0 +1,72 @@
+import { useTranslation } from "react-i18next";
+
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+
+interface DateFieldProps {
+  value?: string;
+  onChange?: (value: string) => void;
+}
```

Comment:
let's also add `disabled: boolean` and `id: string` props too.

`id` would be very much required for cypress tests to recognise and use these inputs.